src/parser/SHOGunApplicationUtil.ts

```diff
@@ -476,6 +478,9 @@ class SHOGunApplicationUtil<T extends Application, S extends Layer> {
     olLayer.set('legendUrl', layer.sourceConfig.legendUrl);
     olLayer.set('hoverable', layer.clientConfig?.hoverable);
     olLayer.set('useBearerToken', layer.sourceConfig?.useBearerToken);
+    if (_isFinite(layer.clientConfig?.opacity)) {
+      olLayer.setOpacity(layer.clientConfig?.opacity ?? 1);
```

Comment:
The opacity is already set in the layer parser methods, e.g. in line 436. Why is this needed twice?

Reply:
the constructor is not working

Reply:
I'll try to find out why

Reply:
Ah nevermind. The opacity has to be set on `layer.clientConfig` instead of directly on the layer

That's why the constructor wasn't working...